nilearn/plotting/html_stat_map.py

```diff
@@ -1,157 +1,494 @@
+        If True, display a colorbar next to the plots.
+    title : string or None (default=None)
+        The title displayed on the figure (or None: no title).
+        This parameter is not currently supported.
+    threshold : str, number or None  (default=1e-6)
+        If None is given, the image is not thresholded.
+        If a string of the form "90%" is given, use the 90-th percentile of
+        the absolute value in the image.
+        If a number is given, it is used to threshold the image:
+        values below the threshold (in absolute value) are plotted
+        as transparent. If auto is given, the threshold is determined
+        magically by analysis of the image.
+    annotate : boolean (default=True)
+        If annotate is True, positions and left/right annotation
+        are added to the plot.
+    draw_cross : boolean (default=True)
+        If draw_cross is True, a cross is drawn on the plot to
+        indicate the cut plosition.
+    black_bg : boolean (default='auto')
+        If True, the background of the image is set to be black.
+        Otherwise, a white background is used.
+        If set to auto, an educated guess is made to find if the background
+        is white or black.
+    cmap : matplotlib colormap, optional
+        The colormap for specified image. The colormap *must* be
+        symmetrical.
     symmetric_cmap : bool, optional (default=True)
-        Make colormap symmetric (ranging from -vmax to vmax).
-        Set it to False if you are plotting an atlas or an anatomical image.
+        True: make colormap symmetric (ranging from -vmax to vmax).
+        False: the colormap will go from the minimum of the volume to vmax.
+        Set it to False if you are plotting a positive volume, e.g. an atlas
+        or an anatomical image.
+    dim : float, 'auto' (default='auto')
+        Dimming factor applied to background image. By default, automatic
+        heuristics are applied based upon the background image intensity.
+        Accepted float values, where a typical scan is between -2 and 2
+        (-2 = increase constrast; 2 = decrease contrast), but larger values
+        can be used for a more pronounced effect. 0 means no dimming.
+    vmax : float, or None (default=None)
+        max value for mapping colors.
+        If vmax is None and symmetric_cmap is True, vmax is the max
+        absolute value of the volume.
+        If vmax is None and symmetric_cmap is False, vmax is the max
+        value of the volume.
+    resampling_interpolation : string, optional (default nearest)
+        The interpolation method for resampling
+        See nilearn.image.resample_img
+    opacity : float in [0,1] (default 1)
+        The level of opacity of the overlay (0: transparent, 1: opaque)
 
     Returns
     -------
-    StatMapView : plot of the stat map.
+    view : the viewer object.
         It can be saved as an html page or rendered (transparently) by the
         Jupyter notebook.
```

Comment:
Can the `view` object returned here be plotted using libraries like `matplotlib`? If so, we can mention that here, something along the lines of `it can be passed as an object to matplotlib.plot()` or something like that.
We can also add an example/single statement of how to save it as HTML using python.

Reply:
There was a discussion about renaming `view_stat_map` to something else, which would indicate its non statistical nature, `view_brain_image` for example. What do you think would be a better name?

Reply:
Thanks! The docs can be updated once we have merged this PR so the changes are set. A lot of documentation is built automatically from docstrings, so once this is completed & merged, then we can rebuild the docs and address the incorrect parts.

Reply:
I've updated the docstring as requested.

Reply:
Thanks!